src/interceptor.js

```diff
@@ -0,0 +1,64 @@
+export default class Interceptor {
+
+  constructor(API, interceptableMethods) {
```

Comment:
Change this to `interceptableMethods = []` so even if it's not passed the `forEach` loop still works.

You may want to add a check if the array is empty and throw an error if need be. Same goes for `API`.